code/game/objects/effects/spawners/faction_spawners.dm

```diff
@@ -0,0 +1,204 @@
+/*
+ * USCM weapons
+ */
+/obj/effect/spawner/random/gun/uscm_primary
+	name = "USCM primary weapon spawner"
+	desc = "spawns USCM primary weapons"
+	spawn_nothing_percentage = 0
```

Comment:
0 by default pretty sure as seen in the parent type

<img width="320" alt="Screen Shot 2023-10-11 at 2 13 01 PM" src="https://github.com/cmss13-devs/cmss13/assets/122310258/6285daaf-5ead-4e3b-b215-6fc7617b5f39">

 Is it necessary to explicitly override this with the same value for uscm_primary, uscm_secondary, and upp? This doesn't seem to be the case for other weapons

<img width="665" alt="Screen Shot 2023-10-11 at 2 19 53 PM" src="https://github.com/cmss13-devs/cmss13/assets/122310258/9b13a5cb-c2d0-44e8-9d2b-6f1590216b13">

Especially since it gets overridden anyway by low chance, high, and med.

Reply:
thanks for catching this